app/scripts/controllers/facebook-reward-controller.js

```diff
@@ -274,6 +274,7 @@ sc.controller('FacebookRewardCtrl', function ($rootScope, $scope, $http, $q, $an
           $scope.reward.updateReward('facebook_error');
           break;
         default:
+          Raven.captureMessage("Unexpected Response: " + response.code);
```

Comment:
We can log the whole `response` like [here](https://github.com/stellar/stellar-client/blob/711f5697ae08b60aceeb0222cfbdec1a8909e9e3/app/scripts/controllers/settings-totp-controller.js#L110):
```js
      Raven.captureMessage('confirmDisableTotp unknown error', {
        extra: {
          error: e
        }
      });
```